ampd/src/stellar/verifier.rs

```diff
@@ -392,4 +418,37 @@ mod test {
                 .unwrap(),
         }
     }
+
+    #[test]
+    fn test_weighted_signers_hash() {
```

Comment:
test should be defined in the file where `hash` is defined and follow a similar pattern of the this [test](https://github.com/axelarnetwork/axelar-amplifier/blob/102d41b76da6c94f9805c51a4779735e4bc7e84f/external-gateways/stellar/src/lib.rs#L447)

Reply:
make sense. updated.